### ISSUE TITLE
debug console.log calls published to npm

### ISSUE THREAD

**Issue by focusaurus** [brianc/node-postgres#947]
Just installed pg@4.2.6 and I'm seeing my stdout polluted with timestamps. I tracked this down to the `lib/utils.js` file in this module. However, the code in git master looks OK, and I couldn't find git history for this code. Thus my theory is whoever did the most recent `npm publish` had this code in their local working directory.

Could you verify and correct with a patch release (not urgent)?

Here's a diff between lib/utils.js from git master and what I got from npm 
```
39c39,44
<       return dateToString(val);
---
>       var x = dateToString(val);
>       console.log(x)
>       console.log(x)
>       console.log(x)
>       return x
```

**Comment by brianc** [comment]
Ah crap! Sorry about that - fixing right now.

**Comment by brianc** [comment]
Fixed on pg@4.5.1!

**Comment by focusaurus** [comment]
Thanks for the quick fix!
